app/presenters/event_presenter.rb

```diff
@@ -0,0 +1,74 @@
+class EventPresenter < BasePresenter
+  presents :event
+
+  delegate :day, :month, :day_of_the_month, :hour, to: :date_presenter
+
+  def speakers_css_class
+    extended_event? ? 'pk-main-speakers--four-speakers' : ''
+  end
+
+  def talks
+    @talk ||= event.talks.includes(:speaker)
```

Comment:
presenter should not query DB. Maybe extract so service

Reply:
done

Reply:
Now this placed in separate finder: [here](https://github.com/pivorakmeetup/pivorak-web-app/pull/707/files#diff-348179df61163682170a18ff069bd798R8).